docs/includes/_access_token_create_conf_client_pkce.mdx

```diff
@@ -1,111 +1,110 @@
-In this flow, you call the app's '/authorize' endpoint, initiating a browser based flow where the user provides their credentials in exchange for a code. 
-In a second API call to the '/token' endpoint, you then exchange the code for an access token. 
-You authenticate the call to the '/token' endpoint with the app's credentials.  
+In this flow, you call the app's '/authorize' endpoint, initiating a browser based flow where the user provides their credentials in exchange for a code.
+In a second API call to the '/token' endpoint, you then exchange the code for an access token.
+You authenticate the call to the '/token' endpoint with the app's credentials.
 
-Note that you will need an identity configured with the [ability to authorize the scopes](/docs/next/api-scopes#scopes-authorized-for-the-identity-approving-the-authorization-request) your app is requesting.   
+Note that you will need an identity configured with the [ability to authorize the scopes](/docs/next/api-scopes#scopes-authorized-for-the-identity-approving-the-authorization-request) your app is requesting.
 
-1. Create an [app](/docs/next/add-an-application) with the following properties:  
+1. Create an [app](/docs/next/add-an-application) with the following properties:
 
-  | Parameter | Value|
-  |---|---|  
-  |Protocol|OIDC|  
-  |Client Type|Confidential|
-  |PKCE|S256|
-  |Token Endpoint Auth Method|Client Secret Basic|
-  |Grant Type|Authorization Code|
-  |Configuration Type (on Authenticator Config tab)|Hosted Web|  
+  | Parameter | Value |
+  | --- | --- |
+  | Protocol | OIDC |
+  | Client Type | Confidential |
+  | PKCE | S256 |
+  | Token Endpoint Auth Method | Client Secret Basic |
+  | Grant Type | Authorization Code |
+  | Configuration Type (on Authenticator Config tab) | Hosted Web |
 
```

Comment:
Nit: Even in these tables we should probably be consistent on what parts are bolded or how they look across each page. Would recommend to follow Next.js

Reply:
Agreed. Bolded the properties. Did a global search, and found all `| Property | Value |` tables are consistent, but these two files were `| Parameter | Value |`. 😆